CONTRIBUTING.md

```diff
@@ -118,6 +118,17 @@ provider_installation {
 
 After installing the necessary software for building provider from sources, you should be able to run `make coverage` to run the tests and see the coverage.
 
+## Package organization for Providers
+We are migrating the resource from SDKv2 to Plugin Framework provider and hence both of them exist in the codebase. For uniform code convention, readability and development, they are defined under internal directory under root as: 
+- `common/provider`: This package contains the changes common to both the providers example provider factory and tests
```

Comment:
There is no `internal/common/provider` package in this PR or on main. Are you planning to introduce this, or is this a stale comment?

Reply:
There is internal/common/provider in the PR:
<img width="297" alt="image" src="https://github.com/user-attachments/assets/bcb2cd7e-a580-44ef-9900-c8d30acdb66b">

Maybe I am getting this wrong, do you mean imports for common provider?
We use that in main.go

`"github.com/databricks/terraform-provider-databricks/internal/common/provider"`